start_ocean.sh

```diff
@@ -32,7 +38,14 @@ show_banner
 
 # default to latest versions
 export OCEAN_VERSION=latest
-COMPOSE_FILE='docker-compose.yml'
+
+COMPOSE_FILES=""
+COMPOSE_FILES+=" -f ${COMPOSE_DIR}/network_volumes.yml"
+COMPOSE_FILES+=" -f ${COMPOSE_DIR}/mongo.yml"
+COMPOSE_FILES+=" -f ${COMPOSE_DIR}/keeper_contracts.yml"
+COMPOSE_FILES+=" -f ${COMPOSE_DIR}/pleuston.yml"
+COMPOSE_FILES+=" -f ${COMPOSE_DIR}/aquarius.yml"
+COMPOSE_FILES+=" -f ${COMPOSE_DIR}/brizo.yml"
```

Comment:
I don't know of the order of these `-f files` matters, but if it does, then pleuston should be last.

Reply:
No, the order does not affect to the running order or any other behavior difference.